charts/prometheus-elasticsearch-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 6.5.1
+version: 6.5.2
```

Please, raise the minor version number.
 ```suggestion
version: 6.6.0
```